src/PlutotvData.cpp

```diff
@@ -113,7 +113,8 @@ bool PlutotvData::LoadChannelsData()
   kodi::Log(ADDON_LOG_DEBUG, "[channels] iterate channels");
   kodi::Log(ADDON_LOG_DEBUG, "[channels] size: %i;", channelsDoc["result"].Size());
 
-  int i = 0;
+  // Use configured start channel number minus one due to the loop logic
+  int i = (GetSettingsStartChannel() - 1);
```

Comment:
Instead of adapting to the loop logic, you could cleanup the loop logic.

```c++
int i = GetSettingsStartChannel();
for (const auto& channel : channelsDoc["result"].GetArray())
{
  ...
  PlutotvChannel plutotv_channel;
  plutotv_channel.iChannelNumber = i++; // position
  ...
}
```